src/components/victory-candlestick/helper-methods.js

```diff
@@ -88,7 +89,7 @@ export default {
     });
   },
 
-  getDomain(props, axis) {
+  getDomainFromData(props, axis) {
```

Comment:
Slightly misleading method name... only the stuff going on in the `else` block is getting the domain from data. Rather than writing a new `getDomain` method, you could also just do something like `let domain;` and then define it in your if / else block, and finally `return Domain.padDomain(domain, props, axis);`